packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -192,44 +176,35 @@ export class RafikiService implements IRafikiService {
       return
     }
 
-    const transferResult = await this.rapydClient.transferLiquidity({
+    await this.gateHubClient.createTransaction({
       amount: this.amountToNumber(amount),
-      currency: amount.assetCode,
-      destination_ewallet: receiverWalletId,
-      source_ewallet: this.env.RAPYD_SETTLEMENT_EWALLET
+      vault_uuid: this.getVaultUuid(amount.assetCode),
+      receiving_address: receiverWallet,
+      sending_address: this.env.GATEHUB_SETTLEMENT_WALLET_ADDRESS,
+      type: 'transfer',
```

Comment:
Shouldn't type be `1` as described [here](https://docs.gatehub.net/api-documentation/c3OPAp5dM191CDAdwyYS/api-reference/appendix)?